doc-examples/azure-cosmos-example-kotlin/src/main/kotlin/example/Child.kt

```diff
@@ -0,0 +1,15 @@
+package example
+
+import io.micronaut.data.annotation.Relation
+import io.micronaut.serde.annotation.Serdeable
+import java.util.ArrayList
+
+@Serdeable
+class Child {
+    var firstName: String? = null
```

Comment:
if you make them data classes you can remove the `?` and null initialization

Reply:
Done